CONTRIBUTING.rst

```diff
@@ -439,7 +439,7 @@ which have different requirements for acceptance into the code base.  These
 requirements are in place for a few reasons -- to make sure that the code is
 maintainable, testable, and that we can easily include information about
 changes in changelogs during the release procedure.  (See `YTEP-0008
-<https://ytep.readthedocs.io/en/latest/YTEPs/YTEP-0008.html>`_ for more
+<https://ytep.readthedocs.io/en/master/YTEPs/YTEP-0008.html>`_ for more
```

Comment:
it's a bit annoying that we need to reference the master branch explicitly, then we can't change the name to `main` without breaking yt's documentation (which we can easily forget about).
Maybe someone with write access to readthedocs could fix it from the inside instead ? (I don't know who that'd be, but I assume @matthewturk or @Xarthisius knows)

Reply:
yes, I agree that this is annoying.  I didn't know how to fix it.

Reply:
second try: @Xarthisius @matthewturk ?

Reply:
Sorry, missed this the first time.  I'll take a look at this.

Reply:
so ... we are rolling with explicit references to the master branch then ?